content/applications/finance/accounting/fiscal_localizations/localizations/mexico.rst

```diff
@@ -64,29 +64,6 @@ The following modules are necessary for all databases that require Mexican local
 #. | **Odoo Mexican localization reports (l10n_mx_reports)**
    | All mandatory reports for electronic accounting. (Requires the accounting application).
 
-The following modules are optional, and should be installed only if they meet a specific
-organization requirement. Installing these modules is not recommended unless you are sure they
-are needed as they add fields that can unnecessarily complicate form filling.
-
-#. | **EDI External Trade Complement for Mexico (l10n_mx_edi_external_trade)**
-   | For clients that export, add the foreign trade complement to the CFDI, and the logic for
-     filling it.
-#. | **Odoo Mexico Localization for Invoice with customs Number (l10n_mx_edi_customs)**
-   | For importing customers, this module allows adding to the CFDI the request number with which
-     the merchandise to be resold entered. When it is imported into Mexico, in the invoice that
-     comes from any foreign country it is necessary to specify which was the import document; This
-     is known as a "pedimento", and it is processed at customs.
-#. | **Odoo Mexico Localization for Stock / Landing (l10n_mx_edi_landing)**
```

Comment:
@raf-odoo we need to keep with this one.
Plus we need to add one extra module that is brand new, below see title and description:
#. | **Odoo Mexican XML Polizas Export (l10n_mx_xml_polizas)**
| With this module, you will be able to export your Journal Entries in XML ready to be uploaded to the SAT.

Reply:
Done, I've removed the reference to the customs module in its description.